pkg/settlement/swap/transaction/transaction_test.go

```diff
@@ -27,6 +27,16 @@ func nonceKey(sender common.Address) string {
 	return fmt.Sprintf("transaction_nonce_%x", sender)
 }
 
+func wrapGasprice(gasprice *big.Int) *big.Int {
+	return transaction.IncreaseGasPrice(gasprice)
+}
+
+func TestGasPrice(t *testing.T) {
+	gasprice := big.NewInt(20)
+	gasprice = transaction.IncreaseGasPrice(gasprice)
+	fmt.Printf("%d", gasprice)
+}
```

Comment:
This is NOT how you write tests in `go` though...

Reply:
just not `assert`, so, what a better way?

Reply:
how about `assert.Equal(t, gasprice, 28)`?